.env.example

```diff
@@ -0,0 +1 @@
+DB_HOST="mongodb+srv://userGoIt:goit@hw03-mongodb.rgoxea7.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
```suggestion
DB_HOST=
```

Reply:
To powinno być bez danych, bo .env.example ma jakie pola ale bez wartości.

Reply:
Dodaj jeszcze 
`PORT=` Bo on też jest w używany

Reply:
Done :)